tools/scripts/fetch.sh

```diff
@@ -12,31 +13,37 @@ set -o nounset
 
 ## Input variables with defaults
 
-
 ## OpenAPI file (latest)
 OPENAPI_FILE_NAME=${OPENAPI_FILE_NAME:-"atlas-api.yaml"}
 
 ## Base URL
-API_BASE_URL=${API_BASE_URL:-"https://cloud.mongodb.com/api/openapi"}
+API_BASE_URL=${API_BASE_URL:-"https://cloud.mongodb.com"}
 
 ## Folder used for fetching files
 OPENAPI_FOLDER=${OPENAPI_FOLDER:-"../openapi"}
-versions_url="$API_BASE_URL/versions"
+
+S3_BUCKET=${S3_BUCKET:-"mongodb-mms-prod-build-server"}
```

Comment:
```suggestion
DATA_KEY=${DATA_KEY:-"mongodb-mms-prod-build-server"}
```

Reply:
I do not see it being used actually.

Reply:
oops good catch, something is up with shellcheck on this file

Reply:
[nit] Can this also be secret: MDB_CURRENT_API_PROD_S3_BUCKET?